PixelToolkit.py

```diff
@@ -109,6 +109,15 @@
         help="Path to a wordlist",
         default="lib/wordlist.txt",
     )
+
+    remote_shell = subparsers.add_parser("remote-shell", help="Remote Shell")
+    remote_shell.add_argument('-c', '--command', action='store_true', help='command shell')
+    remote_shell.add_argument('-e', '--execute', help='execute specified command')
+    remote_shell.add_argument('-l', '--listen', action='store_true', help='listen')
+    remote_shell.add_argument('-p', '--port', type=int, default=5555, help='specified port')
+    remote_shell.add_argument('-t', '--target', default='192.168.0.189', help='specified IP')
```

Comment:
You hardcoded your own local ip address.
Use something like `127.0.0.1` or `localhost` instead

Reply:
Also, I think this option should be named `-b` `--bind` (like in python http server module) or `-a` `--address`